mappings/net/minecraft/world/gen/feature/ClayOreFeature.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_289 net/minecraft/world/gen/feature/ClayOreFeature
```

Comment:
Patches consisting of non-ore blocks should probably be renamed to "blob". So in this case, the class would be called "ClayBlobFeature".

Reply:
I think "ClayVeinFeature" could also work here

Reply:
Seems like "disk_clay" is used more often in Minecraft string constants.